docs/usage/model_config.md

```diff
@@ -389,3 +393,49 @@ except NameError as e:
     print(e)
     #> Field "also_protect_field" has conflict with protected namespace "also_protect_"
 ```
+
+## Hide Input in Errors
+_Pydantic_ shows input value and input type when it raises `ValidationError` during the validation.
+
+
+```py
+from pydantic import BaseModel, ValidationError
+
+
+class Model(BaseModel):
+    a: str
+
+
+try:
+    Model(a=123)
+except ValidationError as e:
+    print(e)
+    """
+    1 validation error for Model
+    a
+      Input should be a valid string [type=string_type, input_value=123, input_type=int]
+    """
+```
+
+You can disable hide the input value and type by setting the `hide_input_in_errors` config to `True`.
```

Comment:
```suggestion
You can hide the input value and type by setting the `hide_input_in_errors` config to `True`.
```

Reply:
yeah, I was about to say `you can disable this behavior` then my mind changed to say `you can hide...` 😄